src/app/api/interest/[interestId]/route.ts

```diff
@@ -0,0 +1,39 @@
+import prismadb from '@/lib/prismadb'
+import { auth } from '@clerk/nextjs/server'
+import { NextResponse, type NextRequest } from 'next/server'
+
+export async function GET(req: NextRequest) {
+  try {
+    const { userId } = auth()
+
+    if (!userId) return new NextResponse('Unauthenticated', { status: 401 })
+
+    // const interests = await prismadb.interest.findUnique({wher})
+
+    // return new NextResponse(JSON.stringify(interests), { status: 200 })
```

Comment:
Will this be used later?

Reply:
Hmm, this probably came from Bernardo's features commits at the dev branch. I'll clean and solve the conflict that appeared 😉